docs/VictoriaLogs/vmalert.md

```diff
@@ -0,0 +1,160 @@
+---
+weight: 10
+title: vmalert
+menu:
+  docs:
+    parent: "victorialogs"
+    weight: 10
+aliases:
+- /VictoriaLogs/vmalert.html
+---
+VictoriaLogs provides log stats APIs [`/select/logsql/stats_query`](https://docs.victoriametrics.com/victorialogs/querying/#querying-log-stats)  and [`/select/logsql/stats_query_range`](https://docs.victoriametrics.com/victorialogs/querying/#querying-log-range-stats), which return the log stats in the format compatible with [Prometheus querying API](https://prometheus.io/docs/prometheus/latest/querying/api/#instant-queries). This allows users using VictoriaLogs as the datasource of [vmalert](https://docs.victoriametrics.com/vmalert/) and creating alerting and recording rules using [LogsQL](https://docs.victoriametrics.com/victorialogs/logsql/).
+
+Note: This page only provides integration instructions for vmalert with VictoriaLogs, see full textbook of vmalert [here](https://docs.victoriametrics.com/vmalert).
+
+## Configuration
+
+Run vmalert with the `-rule.defaultRuleType=vlogs` flag.
```

Comment:
Would be nice to mention that type can be defined on group level as well.

Reply:
Reformated the doc and moved this to `Quick start` section, and want to avoid mentioning mixed datasources there.